docs/howto/custom-rootfs-multiple-targets.md

```diff
@@ -0,0 +1,314 @@
+# How-to deploy a custom rootfs across multiple Windows machines with Landscape API
+
+This guide shows how to use the Landscape API to deploy a custom rootfs
+with some cloud-config user data for further initialisation of the distro
+instances via cloud-init across multiple Windows machines already registered
+with Landscape via Ubuntu Pro for WSL.
+It can be followed either on Windows (using PowerShell) or Linux (using bash).
```

Comment:
I would put these in bullets:

> The steps outlined in this guide can be followed using
> - Bash scripting on Linux
> - PowerShell scripting on Windows

(I guess they could also use PowerShell on Linux but that's probably not relevant here...